ui-tests/src/test/java/io/syndesis/qe/pages/integrations/edit/IntegrationFlowViewComponent.java

```diff
@@ -49,11 +54,11 @@ public String getIntegrationName() {
 	 *
 	 * @param type (start|finish)
 	 */
-	public FlowConnection flowConnection(String type) {
-		SelenideElement stepElement = this.getElementContainingText(Element.STEP, type).shouldBe(visible);
+	public FlowConnection flowConnection(final String type) {
```

Comment:
I'm not sure if we still need FlowConnection class, all we get is still only one element. If we just find element and return it, all will probably work same way. Only difference is isActive method which we don't use in current implementation.

Reply:
@dsimansk just a note: then FlowConnection is redundant.

Reply:
There is exactly one place that we use it :) So I guess I should remove the class and rework the isActive method to `IntegrationFlowViewComponent`?

https://github.com/syndesisio/syndesis-qe/blob/27b8df5c348e01756bdf168c7c0a2909207f657b/ui-tests/src/test/java/io/syndesis/qe/steps/integrations/IntegrationSteps.java#L297